### ISSUE TITLE
Swagger document doesn't work for more that one route parameter

### ISSUE THREAD

**Issue by Natan-Gorecki** [FastEndpoints/FastEndpoints#709]
##  Description
Consider following code:

- Program.cs

```
var builder = WebApplication.CreateBuilder(args);
builder.Services.AddFastEndpoints();
builder.Services.SwaggerDocument();

var app = builder.Build();
app.UseFastEndpoints();
app.UseSwaggerGen();
app.Run();
```

- Update.cs
```
[HttpPut("items/{ParentId}/childs/{Id}")]
[AllowAnonymous]
public class Update : Endpoint<ChildItem, ChildItem>
{
    public override Task HandleAsync(ChildItem req, CancellationToken ct)
    {
        return SendOkAsync(req, ct);
    }
}
public record ParentItem(string Id, string Name, string Description);
public record ChildItem(string Id, string Name, string Description, string ParentId);
```

When I open Swagger UI, the endpoint is discovered and generated. Looks like there is mismatch between PascalCase and CamelCase for the first parameter.
![image](https://github.com/FastEndpoints/FastEndpoints/assets/52935895/5503976c-3bf7-4c17-8c22-66840870afc3)
![image](https://github.com/FastEndpoints/FastEndpoints/assets/52935895/9963d882-79a1-4da3-a35c-d9d558b03ab4)
##  Expected Behavior

- Correct curl is generated
- **Execute** button works, the endpoint returns 200 status and the response body contains parentId with **Value1**

## Actual Behavior

- The curl works from command line, when I replace "{parentid}" with actual value
- Generated curl by Swagger contains "{parentid}" template instead "Value1"
- ParentId value is not passed to the endpoint
- The response contains parentId with **null** value

## Context
System:
```
Operating system: Windows 10 Version 2009
CPU architecture: x86_64
Kernel: winnt 10.0.19045
```

.NET:
```
PS C:\> dotnet --version
8.0.205
```

The NuGet packages used:
```
<PackageReference Include="FastEndpoints" Version="5.25.0" />
<PackageReference Include="FastEndpoints.Swagger" Version="5.25.0" />
```

**Comment by dj-nitehawk** [comment]
thanks for reporting.
this has been fixed in `v5.26.0.10-beta`.

https://github.com/FastEndpoints/FastEndpoints/blob/27d58d164d17520ed61a9c4680cb6183a2d139b8/Src/Library/changelog.md?plain=1#L73-L83

**Comment by Natan-Gorecki** [comment]
Great, thanks for quick response and fix!
